packages/react-scripts/scripts/eject.js

```diff
@@ -45,44 +47,48 @@ prompt(
 
   var folders = [
     'config',
-    path.join('config', 'jest'),
-    'scripts'
+    'config/jest',
+    'scripts',
+    'scripts/utils',
   ];
 
-  var files = [
-    path.join('config', 'env.js'),
-    path.join('config', 'paths.js'),
-    path.join('config', 'polyfills.js'),
-    path.join('config', 'webpack.config.dev.js'),
-    path.join('config', 'webpack.config.prod.js'),
-    path.join('config', 'jest', 'cssTransform.js'),
-    path.join('config', 'jest', 'fileTransform.js'),
-    path.join('scripts', 'build.js'),
-    path.join('scripts', 'start.js'),
-    path.join('scripts', 'test.js')
-  ];
+  // Make shallow array of files paths
+  var files = folders.reduce(function (files, folder) {
```

Comment:
Arrows are fine. If it runs on Node 4 then it's good. But only in `react-scripts`.

Global CLI should stay parseable by Node 0.12 so that we can show a nice error message instead of crashing. (We could also split modern code into a lazy require.)